### ISSUE TITLE
[1.19.2] Crash with Infernal Mobs when using the mob trigger

### ISSUE THREAD

**Issue by EuBruno1** [TheComputerizer/Music-Triggers#164]
When trying to set up the MOB trigger, this crash happens.
Upon further investigation, I found out it is a incompatibility with the infernal mobs mod, as removing the infernal mobs seems to fix do problem.

[crash-2024-03-02_00.53.01-server.txt](https://github.com/TheComputerizer/Music-Triggers/files/14467961/crash-2024-03-02_00.53.01-server.txt)

**Comment by TheComputerizer** [comment]
Yeah seems like I forgot a null check with infernal mobs support. It shouldn't crash if you set the infernal parameter in your mob trigger to `infernal = [ "any" ]` or `infernal = [ "ALL" ]` though

**Comment by 00000000REO** [comment]
Have the exact same problem, can you tell me what exactly I can do to fix it?
It actually does crash even when I add infernal = [ "ALL" ] to all the mob triggers

**Comment by TheComputerizer** [comment]
Fixed in 7.0.0
